src/utils/data.jl

```diff
@@ -54,6 +52,36 @@ function Base.download(
     return data
 end
 
+"""
+Download Data from a "branch" into a "data" folder in given argument path.
+Skip the actual download if the folder already exists and force=false.
+Returns the downloaded folder name.
+"""
+function Base.download(
+    repo::AbstractString,
+    branch::AbstractString,
+    folder::AbstractString,
+    force::Bool = false,
+)
+    if Sys.iswindows()
+        DATA_URL = "$repo/archive/$branch.zip"
```

Comment:
This format may not exist everywhere. You may want to support specific repository names only.